### PR TITLE
Leroy chess

### PR DESCRIPTION
**Pull request by leroy0615**

### CODE REVIEW

**Comment by leroy0615** [comment]
@mesutcan merging my code with master.

File: test/test_helper.rb [hunk 1]
```diff
@@ -1,10 +0,0 @@
-ENV['RAILS_ENV'] ||= 'test'
-require_relative '../config/environment'
-require 'rails/test_help'
-
-class ActiveSupport::TestCase
```
Comment (mesutcan) [inline test/test_helper.rb]: why did you delete this file?

Comment (leroy0615) [reply test/test_helper.rb]: I didn't delete any of those codes. I added Rubocop and set up the database.

**Comment by leroy0615** [comment]
Need to be seen.

**Comment by leroy0615** [comment]
Need to be seen
